Looks like there's a silent merge conflict:

```
kernel/chainparams.cpp: In constructor ‘CMainParams::CMainParams()’:
kernel/chainparams.cpp:194:9: error: no match for ‘operator=’ (operand types are ‘std::__debug::vector<AssumeutxoData>’ and ‘<brace-enclosed initializer list>’)
  194 |         };
      |         ^
In file included from /usr/include/c++/11/vector:76,
                 from /usr/include/c++/11/functional:62,
                 from /usr/include/c++/11/pstl/glue_algorithm_defs.h:13,
                 from /usr/include/c++/11/algorithm:74,
                 from ./span.h:8,
                 from ./uint256.h:10,
                 from ./consensus/params.h:9,
                 from ./kernel/chainparams.h:9,
                 from kernel/chainparams.cpp:6:
/usr/include/c++/11/debug/vector:249:7: note: candidate: ‘std::__debug::vector<_Tp, _Allocator>& std::__debug::vector<_Tp, _Allocator>::operator=(const std::__debug::vector<_Tp, _Allocator>&) [with _Tp = AssumeutxoData; _Allocator = std::allocator<AssumeutxoData>]’
  249 |       operator=(const vector&) = default;
      |       ^~~~~~~~
/usr/include/c++/11/debug/vector:249:17: note:   no known conversion for argument 1 from ‘<brace-enclosed initializer list>’ to ‘const std::__debug::vector<AssumeutxoData>&’
  249 |       operator=(const vector&) = default;
      |                 ^~~~~~~~~~~~~
/usr/include/c++/11/debug/vector:252:7: note: candidate: ‘std::__debug::vector<_Tp, _Allocator>& std::__debug::vector<_Tp, _Allocator>::operator=(std::__debug::vector<_Tp, _Allocator>&&) [with _Tp = AssumeutxoData; _Allocator = std::allocator<AssumeutxoData>]’
  252 |       operator=(vector&&) = default;
      |       ^~~~~~~~
/usr/include/c++/11/debug/vector:252:17: note:   no known conversion for argument 1 from ‘<brace-enclosed initializer list>’ to ‘std::__debug::vector<AssumeutxoData>&&’
  252 |       operator=(vector&&) = default;
      |                 ^~~~~~~~
/usr/include/c++/11/debug/vector:255:7: note: candidate: ‘std::__debug::vector<_Tp, _Allocator>& std::__debug::vector<_Tp, _Allocator>::operator=(std::initializer_list<_Tp>) [with _Tp = AssumeutxoData; _Allocator = std::allocator<AssumeutxoData>]’
  255 |       operator=(initializer_list<value_type> __l)
      |       ^~~~~~~~
/usr/include/c++/11/debug/vector:255:46: note:   no known conversion for argument 1 from ‘<brace-enclosed initializer list>’ to ‘std::initializer_list<AssumeutxoData>’
  255 |       operator=(initializer_list<value_type> __l)
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~
```

--- a/src/kernel/chainparams.cpp
+++ b/src/kernel/chainparams.cpp
@@ -185,7 +185,12 @@ public:
         };
 
         m_assumeutxo_data = {
-            // TODO to be specified in a future patch.
+            {
+                .height = 840'000,
+                .hash_serialized = AssumeutxoHash{uint256{"a2a5521b1b5ab65f67818e5e8eccabb7171a517f9e2382208f77687310768f96"}},
+                .m_chain_tx_count = 991032194,
+                .blockhash = consteval_ctor(uint256{"0000000000000000000320283a032748cef8227873ff4872689bf23f1cda83a5"}),
+            }
         };
 
         chainTxData = ChainTxData{

